fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2,6 +2,9 @@
 //
 // SPDX-License-Identifier: GPL-3.0
 
+/** This class is the top level injection point for many
+ *(if not most?) critical flight parameters.
+ */
```

Comment:
Please don't add this. If anything a comment should describe that this is the legacy FMS and new code should be written in src/fmgc.
```suggestion
```

Reply:
Done